code/modules/mining/lavaland/necropolis_chests.dm

```diff
@@ -670,6 +670,28 @@ GLOBAL_LIST_EMPTY(bloodmen_list)
 	name = "jacob's ladder"
 	desc = "An indestructible celestial ladder that violates the laws of physics."
 
+/obj/item/emberflowers
+	name ="emberflower bouqet"
+	desc ="A charming bunch of flowers, most animals seem to find it very calming after a bit more than a second of contact with it. A tag on the bottom reads 'results not guaranteed with marrow weavers as they lack noses.'"//dont know why marrow weavers keep fucking attacking sometimes
```

Comment:
this is pretty wordy

Reply:
im just now realizing 'a bit more than a second' sounds stupid